### PR TITLE
220 Update dompurify library to latest v2.2.0

### PR DESCRIPTION
**Pull request by oubchid**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "license": "AGPLv3",
   "dependencies": {
     "angularjs-dragula": "2.0.0",
-    "dompurify": "1.0.9",
+    "dompurify": ">=2.2.0",
```
Comment (chibenwa) [inline package.json:19]: Why not using a fixed version?

Comment (MichaelBailly) [reply package.json:19]: Yes correct @chibenwa . I would like that we use the tilde range to specify that we only accept patch level upper versions. https://www.npmjs.com/package/semver#tilde-ranges-123-12-1

**Comment by rezk2ll** [comment]
Why 2 commits?

**Comment by oubchid** [comment]
> Why 2 commits?

I'm using the same process such as gitlab

**Review by rezk2ll — APPROVED** [review]

**Review by rezk2ll — APPROVED** [review]

**Review by MichaelBailly — APPROVED** [review]

**Pull request merged**